### DISCUSSION TITLE
Didn't find upload for firmware and cap file / collab device table.

### DISCUSSION THREAD

**Discussion by Marco-Kl** [ct-Open-Source/tuya-convert#991] [Q&A]
Hello,
maybe it is to late today or I am blind. Where or how can i upload my dumped Tuya firmware and tcpdump of Gosund EP2 plug communication?

Best regards Marco

**Comment by kueblc** [comment]
Hi @Marco-Kl 

Thanks for considering donating your firmware and tcp dumps! You can zip and attach them directly to a github comment.
